build/scripts/cleanup_perf.ps1

```diff
@@ -13,25 +16,49 @@ try {
 }
 catch {}
 try {
+    taskkill /F /IM csc.exe 2>&1 | Out-Null
+}
+catch {}
+try {
+    taskkill /F /IM vbc.exe 2>&1 | Out-Null
+}
+catch {}
+try {
     taskkill /F /IM VBCSCompiler.exe 2>&1 | Out-Null
 }
 catch {}
 
-# Move all etl files to the a folder for archiving
-echo "creating ToArchive directory"
-mkdir ToArchive
-echo "moving C:/CPC/DataBackup* to ToArchive"
-mv C:/CPC/DataBackup* ToArchive
-ls ToArchive
+try {
+    & $CPCLocation/cpc.exe /Stop /SkipClean 2>&1 | Out-Null
+}
+catch {}
+
+If (Test-Path ToArchive) {
+    Remove-Item -Recurse -Force ToArchive
+}
+
+If (Test-Path $CPCLocation) {
```

Comment:
Use `if` instead of `If` to be consistent with other casing decisions in this script and others